src/pages/account.tsx

```diff
@@ -1,30 +1,181 @@
+    // gather the data from the form
+    const formData = { email: email , name: username, type: accountType };
+
+    // send the data to the server TO DO: create API route
+    const response = await fetch('/api/user/updateUser', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(formData),
+    });
+    console.log(formData)
+    console.log(response);
+
+    // update the user's session
+    
+  };
+
+  const handleCancelClick = () => {
+    setIsEditing(false);
+  };
 
-  // const isNewUser = user.type === 'new';
+  const name = user?.name || user?.email || 'Rodeo Co User';
+  const email = user?.email;
+  const avatarLetter = name[0].toUpperCase();
+  const avatarSrc = user?.image;
+  const avatarSize = 100;
+
+  let userType;
+  if (user?.type === 'new') {
+    userType = 'New';
+  } else if (user?.type === 'admin') {
+    userType = 'Producer';
+  } else {
+    userType = 'Participant';
+  }
 
-  // todo: if user.type === 'new', make the user choose admin or participant type
-  // ^ this should probably be done in middleware, actually - a forced redirection to /account (this view) until user has a real type
   return (
-    <p>Are you a rodeo administrator looking to create rodeos, or a participant?</p>
-  )
+    <>
+      <Box p={4} sx={{fontFamily:'Poppins, sans-serif'}}>
+        <Box display="flex" alignItems="top" mb={2} sx={{fontFamily:'Poppins, sans-serif'}}>
+          <Avatar
+            sx={{
+              bgcolor: '#3f51b5',
+              fontSize: '6rem',
+              width: avatarSize,
+              height: avatarSize,
+              mr: 2,
+            }}
+            alt={name}
+            src={avatarSrc}
+          >
+            {avatarSrc ? null : avatarLetter}
+          </Avatar>
+          <Box>
+            <Typography variant="h4" gutterBottom sx={{fontFamily:'Poppins, sans-serif'}}>
+              {name}
+            </Typography>
+            <Typography variant="subtitle1" gutterBottom sx={{fontFamily:'inherit'}}>
+              Rodeo {userType} 
+            </Typography>
+            <Typography variant="subtitle1" gutterBottom sx={{fontFamily:'inherit'}}>
+              {user?.email}
+            </Typography>
+            <Button
+              variant="outlined"
+              color="primary"
+              onClick={handleEditClick}
+              sx={{
+                color: '#CF7F49',
+                borderColor: '#CF7F49',
+                '&:hover': {
+                  backgroundColor: '#CF7F49',
+                  color: '#fff',
+                  borderColor: '#CF7F49',
+                },
+                borderRadius: '25px',
+                fontFamily: 'inherit',
+              }}
+
+            >
+              Edit Profile
+            </Button>
+          </Box>
+
+          {/* Edit Profile Modal  */}
+            <Dialog open={isEditing} onClose={handleCancelClick} sx={{fontFamily: 'Poppins, sans-serif'}}>
```

Comment:
There's also an `OpenModalButton` component that abstracts all the onClose() if you want to use it. I probably would, but I think it's fine if you don't. Just bringing it up if you are breaking the form into another component anyway

Reply:
That's a great idea but in the interest of time I'm going to put that off for later